main.rb

```diff
@@ -0,0 +1,18 @@
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
+          
+        end
+end
+def foo
```

Comment:
__[development][rubocop]__
* Method `Object#foo` is defined at both /sideci/sideci_rails_sandbox/main.rb:2 and /sideci/sideci_rails_sandbox/main.rb:10.

<!-- Comment by [SideCI](https://sideci.com) -->